python/python/bystro/search/index/listener.py

```diff
@@ -80,8 +80,8 @@ def run_handler_with_config(
         mapping_config,
         "--opensearch-config",
         opensearch_config,
-        "--tarball-path",
-        tar_path,
+        "--input",
```

Comment:
Just a clarification question - if the tarball-path is the output, shouldn't this be --output instead of --input? Or is this the vcf path that's the input and it's also the same path for the annotation output?

Reply:
Ah, good question, it is confusing; here we're calling the bystro opensearch program, which takes an annotation as an input. Previously it took the tarball as input (since the tarball contained the annotation)

Reply:
Ok got it!